gulpfile.js

```diff
@@ -30,7 +31,23 @@ var paths = {
 		'node_modules/select2/dist/css/select2.min.css',
 		'node_modules/select2/dist/js/select2.full.js',
 		'node_modules/select2/dist/js/select2.full.min.js'
-	]
+	],
+	packageContents: [
+		'assets/**/*',
+		'changelog.txt',
+		'CONTRIBUTING.md',
+		'dummy_data.xml',
+		'includes/**/*',
+		'lang/**/*',
+		'README.md',
+		'templates/**/*',
+		'uninstall.php',
+		'widgets/**/*',
+		'woothemes-sensei.php',
+		'wpml-config.xml',
```

Comment:
Should we also include `LICENSE`?

Reply:
One of the requirements of most licenses is that they are included with the distribution, so yes :-)

Reply:
Good catch! Added `LICENSE` to the list 🙂